apps/wizard/app_pages/chart_diff/chart_diff_show.py

```diff
@@ -157,6 +157,7 @@ def _push_status_binary(self, session: Optional[Session] = None) -> None:
 
     def _pull_latest_chart(self):
         """Get latest chart version from database."""
+        # TODO: can we use ChartDiffLoader for this and get rid of `from_chart_id`?
```

Comment:
Yeah, totally. It'd be also nice to merge some of the logic from `ChartDiff.from_chart_id` with `ChartDiff.from_charts_df`

Reply:
Done!